examples/anthropic/tracing.py

```diff
@@ -0,0 +1,27 @@
+"""
+This is an example for leveraging MLflow's auto tracing capabilities for Anthropic.
+
+For more information about MLflow Tracing, see: https://mlflow.org/docs/latest/llms/tracing/index.html
+"""
+
+import os
+
+import mlflow
+
+# Turn on auto tracing for Anthropic by calling mlflow.anthropic.autolog()
+mlflow.anthropic.autolog()
+
+# Import the SDK and configure your API key.
+import anthropic
```

Comment:
```suggestion
import anthropic
import mlflow
import os

# Turn on auto tracing for Anthropic by calling mlflow.anthropic.autolog()
mlflow.anthropic.autolog()

```

nit: to avoid mislead that `mlflow.anthropic.autolog()` needs to be called before importing Anthropic SDK

Reply:
sure!